locations/spiders/gov_goriva_si.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any, Iterable
+
+from scrapy import Request, Spider
+from scrapy.http import JsonRequest, Response
+
+from locations.categories import Categories, apply_category, apply_yes_no
+from locations.dict_parser import DictParser
+
+# https://www.gov.si/novice/2020-09-28-spletna-aplikacija-goriva-si/
+
+
+class GovGorivaSISpider(Spider):
+    name = "gov_goriva_si"
+    brands = {
+        1: {"brand": "Petrol", "brand_wikidata": "Q174824"},
+        3: {"brand": "MAXEN"},
+        4: {"brand": "MOL", "brand_wikidata": "Q549181"},
+        5: {"name": "Shell", "brand": "Shell", "brand_wikidata": "Q110716465"},
+    }
+    fuels = {
+        "95": "octane_95",
+        "dizel": "diesel",
+        "98": "octane_98",
+        "100": "octane_100",
+        "dizel-premium": None,
+        "avtoplin-lpg": None,
```

Comment:
According to https://wiki.openstreetmap.org/wiki/Key:fuel this can be mapped to `lpg`:
````suggestion
        "avtoplin-lpg": "lpg",
````